DQM/HLTEvF/plugins/HLTObjectsMonitor.cc

```diff
@@ -345,15 +347,20 @@ HLTObjectsMonitor::analyze(const edm::Event& iEvent, const edm::EventSetup& iSet
 
   // loop over path
   int ibin = -1;
+  std::unordered_set<int> plottedPathNameIndices;
```

Comment:
I suspect that using a `vector<bool>` with the bin as the index, instead of an `unordered_set<int>` to keep track of which bins have been filled would be faster ans use less memory.